src/app/messages/en/messages.json

```diff
@@ -45,7 +45,8 @@
   "i18n:footer:social:rice-logo:alt": "Rice University logo",
   "i18n:search-results:bar": "Search results sidebar",
   "i18n:search-results:bar:loading-state": "Loading results",
-  "i18n:search-results:bar:query:results": "search results for ",
+  "i18n:search-results:bar:query:multi-results": "search results for ",
+  "i18n:search-results:bar:query:one-result": "search result for ",
```

Comment:
there are formatting options with params you can use for this pluralization, look at the docs for react-intl